internal/pkg/template/template.go

```diff
@@ -54,6 +56,15 @@ func LoadInputs(fs filesystem.Fs) (StepsGroups, error) {
 	return templateInput.Load(fs)
 }
 
+func LoadReadme(fs filesystem.Fs) (string, error) {
+	path := filesystem.Join(SrcDirectory, ReadmeFile)
+	file, err := fs.ReadFile(filesystem.NewFileDef(path).SetDescription("template readme"))
+	if err != nil {
+		return "", err
+	}
+	return file.Content, nil
+}
+
```

Comment:
Tu su 2 zmeny:
1. README sa doteraz vobec nenacitavalo, teraz sa nacitava
2. `README.md` sa ukladalo do `template/README.md`, teraz sa uklada do `template/src/README.md` ... pretoze, v CLI stahujeme z Git-u iba `src` adresar, takze bez toho by to neslo.